extensions/kube-object-event-status/package-lock.json

```diff
@@ -16,9 +16,9 @@
       },
       "dependencies": {
         "@babel/runtime": {
-          "version": "7.14.0",
-          "resolved": "https://registry.npmjs.org/@babel/runtime/-/runtime-7.14.0.tgz",
-          "integrity": "sha512-JELkvo/DlpNdJ7dlyw/eY7E0suy5i5GQH+Vlxaq1nsNJ+H7f4Vtv3jMeCEgRhZZQFXTjldYfQgv2qmM6M1v5wA==",
+          "version": "7.16.0",
```

Comment:
Idk why package-lock files within extensions were changed but these changes should not be part of this PR.

Reply:
True, removed.